conda_build_config.yaml

```diff
@@ -108,14 +108,16 @@ icu:
   - 73
 jpeg:
   - 9
+libcurl:
+  - 8.1.1
```

Comment:
We update curl/libcurl relatively often. Is it fine to place it here and pin it so strictly?

Reply:
Yes, libcurl has a run_exports on the major only. `libcurl {{ libcurl }}` in host will yield the broadest range `>=8.1.1,<9`.